latest/index.bs

```diff
@@ -331,24 +338,1010 @@ Conforming readers:
+
+If part of [[#multiscale-md]], the length of "axes" MUST be equal to the number of dimensions of the arrays that contain the image data.
+
+<div class=example>
+
+Examples of valid axes:
+
+```json
+[
+    {"name": "x", "type": "space", "unit": "micrometer"},
+    {"name": "t", "type": "time", "unit": "second", "longName": "Unix Epoch time"},
+    {"name": "c", "type": "channel", "discrete": true},
+    {"name": "i0", "type": "array"},
+    {"name": "c", "type": "coordinate", "discrete" : true },
+    {"name": "v", "type": "displacement", "discrete": true },
+    {"name": "freq", "type": "frequency", "unit": "megahertz"}
+]
+```
+</div>
+
+Arrays are often thought of as containing discrete samples along the continuous variable. Axes representing space and time are
+usually continuous, meaning they can be indexed by real-valued (floating point) numbers whereas discrete axes may be indexed
+only by integers. Arrays are inherently discrete (see Array coordinate systems, below) , but values "in between" discrete
+coordinates can be retreived using an *interpolation* method.  If an axis is continuous (`"discrete" : false`), it indicates
+that indexing with continuous values is meaningful, and that interpolation may be performed along that axis. Interpolation may
+be performed jointly across axes with the same `type`, and interpolation should not be performed for discrete axes or jointly
+across axes with differing `type`s.  Other non-continuous variables and axis types are also usually discrete, such as
+`channel`s, `coordinate`s, and `displacement`s.
+
+Note: The most common methods for interpolation are "nearest neighbor", "linear", "cubic", and "windowed sinc". Here, we refer
+to any method that obtains values at real valued coordinates using discrete samples as an "interpolator". As such, label images
+may be interpolated using "nearest neighbor" to obtain labels at points along the continuum.
+
+<div class=example>
+
+For the coordinate system:
+
+```json
+{
+    "name" : "index and interpolation",
+    "axes" : [
+        {"name": "c", "type": "channel", "discrete": true},
+        {"name": "x", "type": "space"},
+        {"name": "y", "type": "space"},
+        {"name": "t", "type": "time"},
+    ]
+}
+```
+
+Indexing an image at the point `(0.1, 0.2, 0.3, 0.4)` is not valid, because the value of the first coordinate (`0.1`) refers
+to the discrete axis `"c"`.  Indexing an image at the point `(1, 0.2, 0.3, 0.4)` is valid.
+</div>
+
+The axes names of an array MAY be duplicated in the Zarr metadata key `dimension_names`. 
+Implementations MUST give precedence to the `axes` metadata over the `dimenions_names`.
+
+
+### Array coordinate systems
+
+Every array has a default coordinate system whose parameters need not be explicitly defined.  Its name is the path to the array
+in the container, its axes have `"type":"array"`, are unitless, and have default "name"s. The ith axis has `"name":"dim_i"`
```

Comment:
Shouldn't the default names be those specified by the zarr v3 `dimension_names` property?

Reply:
I see this is kind of addressed in the paragraph above, but it still might be helpful to clarify here.